CONTRIBUTING.md

```diff
@@ -419,19 +419,45 @@ The Chapter client uses the React framework [Next.js](https://nextjs.org/) with
 After you have added new feature, to make sure it stays working, we recommend using [Cypress](https://www.cypress.io/). It will automatically test different scenarios starting in the client side of the application, communicating with the server, and warning you if something unexpected happens.
 </details>
 
-## Where to Find the Code
+## Where to Find the Code to Change
 
+### Database schema
 * The database schema is defined in _server/prisma/schema.prisma_
+
+<details><summary><b>Making schema changes</b></summary>
+
+* Schema changes require recreating of the database, this is done with `npm run db:reset` command.
+* After making changes to schema it might be needed to change the way sample data is generated in `server/prisma/generator` folder.
+
+</details>
+
+### Server
 * GraphQL object types are defined by files in _server/src/graphql-types_
 * Resolvers for the GraphQL queries are defined in _server/src/controllers_
+
+<details><summary><b>Details</b></summary>
+
+* GrapQL object type define object cooresponding to the table in database.  It can be used by mutations and queries as an input or return type.  More complicated tables can be modelled using multiple GraphQL objects.
+* Resolver contains the logic required for responding to mutation or query requests send from client.
+* New resolver needs to be exported in the `server/src/controllers/index.ts` file.
+
+</details>
+
+### Client
 * The client accesses the data via hooks defined in _client/src/generated/generated.tsx_
 * To create new hooks, modify _queries.ts_ and _mutations.ts_ files in _client/src/modules/**/graphql_
 * Client pages are defined according to [Next.js's routing](https://nextjs.org/docs/routing/dynamic-routes) e.g. _client/src/pages/dashboard/events/\[id\]/edit.tsx_ handles pages like _/dashboard/events/1/edit_
 * Cypress test coverage spec files should go in _/cypress/e2e_, roughly mirroring the client pages pattern
 
-## Where to Find the Issues
+<details><summary><b>Details</b></summary>
```

Comment:
```suggestion
<details><summary><b>Linking the client to the schema</b></summary>
```

Reply:
Yeah, what would be making the additions not related to _just_ where to find the code.

Reply:
It does 🤔, but do we mind? People who are coming to help shouldn't be expecting us to tell them how the code function, we aren't building GraphQL or Prisma 2nd docs. 

We are telling them hey "this is how we link it, in these files you will see our code, if you want to know more about it, we expect you to open the docs".